sycl/source/device_selector.cpp

```diff
@@ -72,6 +72,10 @@ static void traceDeviceSelection(const device &Device, int Score, bool Chosen) {
 
 device select_device(DSelectorInvocableType DeviceSelectorInvocable,
                      std::vector<device> &Devices) {
+  if (Devices.empty()) {
+    std::cerr << "Warning: No devices satisfy the currently set filter!"
```

Comment:
Should we generate this message only in the case when ONEAPI_DEVICE_SELECTOR is set?

@cperkinsintel could you please provide your input as you were contributing a lot to selectors lately?

Reply:
@lbushi25  - I'm not sure this is the right place to emit this warning. What if there are no devices present at all?  Also, this could end up emitting lots of warnings not just one.